python/palletjack/palletjack.cc

```diff
@@ -588,26 +594,29 @@ std::shared_ptr<parquet::FileMetaData> ReadMetadata(const DataHeader &dataHeader
         }
     }
 
-    index_src = row_groups_offsets[dataHeader.get_row_groups_offsets_size()];
+    index_src = row_groups_offsets[1 + dataHeader.row_groups];
```

Comment:
It was off by one error introduced by https://github.com/marcin-krystianc/PalletJack/pull/45/files#diff-78e4256a916c766f7d02bf4eab9d47903dc088a4c955e760088a5d60ae1e2b63R591